### DISCUSSION TITLE
[BUG]Not receiving XCH rewards -272765

### DISCUSSION THREAD

**Discussion by 13430662839** [Chia-Network/chia-blockchain#5140] [Support]
I dug 2XCH at altitude 272765 ，But I haven't received any XCH rewards yet。

Check in chiaexplorer  that the reward has been issued，But my address did not prompt the account 。

Block confirmation height is 272765，Reward confirmation height is 272768，
Reward address ：xch1eg3pmlvqaj4p5228kaqed6dlrhg49hc8gc9t0qayckmfug9qduvqp0gnf2 
 This Reward address is apply from  https://www.mxc.la/ 。

Search  272765 at  chiaexplorer ，You can query the records of farming rewards。
![image](https://user-images.githubusercontent.com/84167842/118346962-1c433d80-b572-11eb-84a3-4f0da6a272c3.png)

Search  272768 at  chiaexplorer ，You can query the record of the award。
![image](https://user-images.githubusercontent.com/84167842/118347006-6f1cf500-b572-11eb-9e9f-7029d57a37ca.png)

Search  xch1eg3pmlvqaj4p5228kaqed6dlrhg49hc8gc9t0qayckmfug9qduvqp0gnf2  at  chiaexplorer ，Check that the reward has been distributed。
![image](https://user-images.githubusercontent.com/84167842/118347043-b6a38100-b572-11eb-9656-f4519f5feb4a.png)

But in my MXC_XCH check did not receive a reward。
![image](https://user-images.githubusercontent.com/84167842/118347065-e9e61000-b572-11eb-8a7d-c41b8bc9ced9.png)

**Comment by Jufik** [comment]
@13430662839 I'm no expert but I'd say if your block is out there:

- Wait for like.... few minutes to let your plateform update the balance
- Contact your platform if it doesn't show.

BTW, is your balance still empty?

**Comment by 13430662839** [comment]
I have wait two days。


My balance is still&nbsp; empty 。


Then I have to go to the continuous platform。









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Chia-Network/chia-blockchain"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月15日(星期六) 下午3:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Chia-Network/chia-blockchain] [BUG]Not receiving XCH rewards -272765 (#5058)





 
@13430662839 I'm no expert but I'd say if your block is out there:
  
Wait for like.... few minutes to let your plateform update the balance
 
Contact your platform if it doesn't show.
  
BTW, is your balance still empty?

**Comment by lvcivs** [comment]
Are you solo farming or in a pool?
Is your wallet synced?

**Reply by sargonas** [reply]
To add to this, I can confirm looking a the block I also see you got the reward. If anything, sounds like your wallet is out of sync locally which is why you don't see it. Can you confirm what block height your wallet is at, as well as your local node (and the date and time that you checked it)? Comparing those I can give you a rough idea if you are in sync or behind.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
